taf/testlib/linux/collectd.py

```diff
@@ -34,160 +34,99 @@
 env.lhost[1].ui.collectd.start()
 env.lhost[1].ui.collectd.stop()
 env.lhost[1].ui.collectd.restart()
-
-Example of collectd.conf modifications:
-format: instance.ui.collectd.<action>.<plugin_name>()
-env.lhost[1].ui.collectd.enable.csv(DataDir='"/path/to/logs"')
-env.lhost[1].ui.collectd.disable.csv()
-env.lhost[1].ui.collectd.enable_defaults.csv()
-env.lhost[1].ui.collectd.change_param.csv(DataDir='"/path/to/logs"')
-env.lhost[1].ui.collectd.insert_param.csv(DataDir='"/path/to/logs"')
+env.lhost[1].ui.collectd.update_config_file()
```

Comment:
Can you please provide complete example for collectd.conf update.
e.g.:
env.lhost[1].ui.global_options = {'Interval': 1}
env.lhost[1].ui.collectd.loadplugin_tags = {'csv': {}}
env.lhost[1].ui.collectd.loadplugin_tags = {'csv': {}}
env.lhost[1].ui.collectd.plugins_options = {'csv': {'DataDir': '"/path/to/some/dir"'}}
env.lhost[1].ui.collectd.update_config_file()